server/operator.go

```diff
@@ -121,6 +122,11 @@ func (bo *balanceOperator) Do(ctx *opContext, region *regionInfo) (bool, *pdpb.R
 		return true, nil, nil
 	}
 
+	if !bo.Started {
+		bo.Started = true
+		bo.Start = time.Now()
```

Comment:
can we use Start only, if Start > 0, we can think the operator is started. And we will set Start to 0 after we finish the operator.